src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -145,12 +145,14 @@ export default class MeetingSessionConfiguration {
       }
       this.meetingId = createMeetingResponse.meetingid;
       this.urls = new MeetingSessionURLs();
-      this.urls.audioHostURL = createMeetingResponse.mediaplacement.audiohosturl;
-      this.urls.screenDataURL = createMeetingResponse.mediaplacement.screendataurl;
-      this.urls.screenSharingURL = createMeetingResponse.mediaplacement.screensharingurl;
-      this.urls.screenViewingURL = createMeetingResponse.mediaplacement.screenviewingurl;
-      this.urls.signalingURL = createMeetingResponse.mediaplacement.signalingurl;
-      this.urls.turnControlURL = createMeetingResponse.mediaplacement.turncontrolurl;
+      if (createMeetingResponse.mediaplacement !== null) {
```

Comment:
Note that this `if` condition can be `true` when `createMeetingResponse.mediaplacement` is `undefined`, any number, and any string. Then, accessing properties on those primitives will fail.